ballerina/tests/utils_test.bal

```diff
@@ -0,0 +1,205 @@
+import ballerina/test;
+
+//import ballerina/io;
+
+@test:Config {}
+function testGetDeepObjectStyleRequest() {
```

Comment:
I think we don't need to write test cases for util functions as they were generated from the code. Hence, lets remove this file altogether.